packages/Metric/src/Score.tsx

```diff
@@ -36,6 +38,35 @@ const Score: React.FunctionComponent<ScoreProps> = ({
 }: ScoreProps) => {
     const stringFormatter = useLocalizedStringFormatter(intlMessages);
     const { locale } = useLocale();
+
+    const getBrand = (): string => {
+        return document.documentElement.getAttribute("data-brand") ?? "igloo";
+    };
+
+    const isWorkleap = getBrand() === "workleap";
+
+    const sizeMap = {
+        small: "sm",
+        medium: "md",
+        large: "lg"
+    };
+
+    const arrowPositiveClass = cx("ids-score__arrow", "ids-score__arrow--positive", {
+        "ids-score__arrow--selected": isSelected
+    });
+
+    const arrowNegativeClass = cx("ids-score__arrow", "ids-score__arrow--negative", {
+        "ids-score__arrow--selected": isSelected
+    });
+
+    const ArrowUpIconElement = isWorkleap ?
+        <ArrowUpIcon className={arrowPositiveClass} size={sizeMap[arrowSize] as IconProps["size"]} /> :
```

Comment:
this does work but I meant more
`const ArrowUpIconComponent = isWorkleap ? ArrowUpIcon : ArrowUp;`
Then this could be used directly in the component.